server/lib/src/idm/oauth2.rs

```diff
@@ -3034,7 +3039,7 @@ mod tests {
                 response_type: ResponseType::Code,
                 response_mode: None,
```

Comment:
Can we update one of the tests to include the `None` case just to be sure it's covered please 🥺